libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -111,8 +112,10 @@ void AP_Frsky_Telem::send_SPort_Passthrough(void)
                 _passthrough.params_timer = AP_HAL::millis();
                 return;
             } else if ((now - _passthrough.ap_status_timer) > 500) {
-                send_uint32(DIY_FIRST_ID+1, calc_ap_status());
-                _passthrough.ap_status_timer = AP_HAL::millis();
+                if ((*_ap.valuep & AP_INITIALIZED_FLAG) > 0) {  // send ap status only once vehicle has been initialised
```

Comment:
Thanks Tom for looking at this PR. I'm anding the value pointed to (*), just like it is done in the calc_ap_status() function of the same file...